base/redactor_userdata_test.go

```diff
@@ -73,3 +74,84 @@ func BenchmarkUserDataRedact(b *testing.B) {
 		}
 	})
 }
+
+type FakeLogger struct {
+}
+
+func (fakeLogger FakeLogger) String() string {
+	time.Sleep(10 * time.Millisecond)
```

Comment:
As mentioned in slack, would be good to lower this to reduce the artificial differences in benchmarking.

```suggestion
	time.Sleep(10 * time.Microsecond)
```